### DISCUSSION TITLE
Are you having any issues running the simulator? - Jan 6

### DISCUSSION THREAD

**Discussion by ankushdharkar** [RealDevSquad/Centauri-React-Native-Cohort#6] [Q&A]
Post your issues here and we shall try to get them fixed

**Comment by vinaybajjuri58** [comment]
Hey Ankush 
Im facing and issue in running the simulator
I tried my best in resolving this issues but I was not able to and I have also posted in the discussions 
npx react-native start command is working fine and metro is getting started 
once I execute npx react-native run-android command AVD was launched then comes this issue of ` java.net.ConnectException: Connection refused: connect `
![image](https://user-images.githubusercontent.com/57398305/148510373-9103cc22-e765-4084-804f-73d06cf4db6c.png)
![image](https://user-images.githubusercontent.com/57398305/148510412-0fdece30-7a45-4c0e-85ec-04ad6e58dd8c.png)
![image](https://user-images.githubusercontent.com/57398305/148510440-a0fd2112-9fc0-4bdd-9bc3-c054b36aeaf9.png)

Can you please help me in resolving this issue

**Reply by harshil28** [reply]
I am also having the same issue...can anybody help..?

**Reply by vinaybajjuri58** [reply]
@harshil28 Were you able to resolve the issue ?

**Comment by viveks221** [comment]
![env](https://user-images.githubusercontent.com/56931778/148571234-8b0e3166-8a1f-4a1d-85be-6dccde189c4b.PNG)
  
**have you setup the environment variable??**

**Reply by vinaybajjuri58** [reply]
Yes I have done that
But still facing the same issue

**Reply by chaitanyaj2222** [reply]
You might have missed out on a few steps while you were installing android studio. I faced the same issue earlier & I tried to re-install the whole android studio **carefully** by following the steps in [this link](https://reactnative.dev/docs/environment-setup). 

Kindly get back here so if the issue is still persistent. We can try to figure it out, together!

**Reply by vinaybajjuri58** [reply]
Oh okay @Chaitanya7666 
I'll try again and let you know here
Thanks for the response
